sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -934,8 +934,7 @@ def validate(self, validator):
     errors.extend(validator.validate_sdk_container_image_options(self))
 
     if validator.is_service_runner():
-      errors.extend(
-          validator.validate_optional_argument_positive(self, 'num_workers'))
+      errors.extend(validator.validate_worker_num(self))
```

Comment:
Nit: function name should be consistent with option names where possible. Helps with readability as well as grep.
```suggestion
      errors.extend(validator.validate_num_workers(self))
```

Reply:
Ok. Fixed this and all the other similar requests.